app/DoctrineMigrations/Version20190301182150.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
```

Comment:
Are we going to declare this for all new php files? Now, that we have moved to PHP7

Reply:
Personally I think it would be a good direction, especially for all new code.